### PR TITLE
[15.0][FIX] sale_financial_risk: call the right methods

### PR DESCRIPTION
**Pull request by chienandalu**
cc @Tecnativa TT45035

please review @sergio-teruel @carlosdauden

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
A regression test would be very advisable.

**Review by sergio-teruel — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Comment by chienandalu** [comment]
> A regression test would be very advisable.

Done

File: sale_financial_risk/tests/test_payment_financial_risk.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+
+@tagged("-at_install", "post_install")
+class TestRiskSalePayment(PaymentCommon, PaymentHttpCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(
+            context=dict(cls.env.context, tracking_disable=True, mail_create_nolog=True)
+        )
```
Comment (pedrobaeza) [inline sale_financial_risk/tests/test_payment_financial_risk.py]: I have this more complete one for your reference:

```suggestion
        cls.env = cls.env(
            context=dict(
                cls.env.context,
                mail_create_nolog=True,
                mail_create_nosubscribe=True,
                mail_notrack=True,
                no_reset_password=True,
                tracking_disable=True,
            )
        )
```

**Review by pedrobaeza — COMMENTED** [review]
Thanks for the test

/ocabot merge patch

File: sale_financial_risk/tests/test_payment_financial_risk.py (hunk 1 above)
Comment (chienandalu) [reply sale_financial_risk/tests/test_payment_financial_risk.py]: Thanks :) Included

**Comment by pedrobaeza** [comment]
Agh, too late for the latest change, but it was not critical.

**Comment by chienandalu** [comment]
hehehe

**Pull request closed without merging**